pinc/Activity.inc

```diff
@@ -6,6 +6,8 @@ include_once($relPath.'quizzes.inc'); // get_Quiz_with_id
 include_once($relPath.'misc.inc'); // startswith
 include_once($relPath.'User.inc');
 
+// this declaration is needed to enable phpunit to work correctly
```

Comment:
I'd remove these comments (here and in `ProjectTransition.inc`). Because while they are accurate in this case, we should be more generally declaring global variables as global because otherwise they are just globals by accident if the file is included in the global scope.

I'd also move this right above where `$ACCESS_CRITERIA` is defined a few lines below.

Reply:
Done this.